pkgs/applications/video/mediathekview/default.nix

```diff
@@ -1,33 +1,35 @@
-{ lib, stdenv, fetchurl, makeWrapper, jre, zip }:
+{ lib, stdenv, fetchurl, makeWrapper, libglvnd, libnotify, jre, zip }:
 
 stdenv.mkDerivation rec {
-  version = "13.8.0";
+  version = "13.9.1";
   pname = "mediathekview";
   src = fetchurl {
     url = "https://download.mediathekview.de/stabil/MediathekView-${version}-linux.tar.gz";
-    sha256 = "0zfkwz5psv7m0881ykgqrxwjhadg39c55aj2wpy7m1jdara86c5q";
+    sha256 = "4BYKkYhl1YjiAZyfNRdV5KQL+dVkL058uhTG892mXUM=";
   };
 
+  buildInputs = [ libglvnd libnotify ];
```

Comment:
We shouldn't use building it's when using it in scripts below because I don't think that will be spliced correct anyway

Reply:
Good point, I've removed it.